cylc/flow/unicode_rules.py

```diff
@@ -27,7 +27,7 @@
 from cylc.flow.task_state import TASK_STATUSES_ORDERED
 
 ENGLISH_REGEX_MAP = {
-    r'\w': 'alphanumeric',
+    r'\w': 'alphanumeric (defined by ``str.isalnum()``) or ``_``',
```

Comment:
Technically correct, but a tad cryptic!

Not sure how to communicate this clearly, perhaps:

```
alphanumeric (regex word characters - `\w+`)
```